<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 45px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>600</y>
     <width>800</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/pictures/background/Untitled-2.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="deal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 249, 202);
  background:qlineargradient(spread:pad, x1:0.523, y1:0.932, x2:0.522818, y2:0.136, stop:0 rgba(0, 134, 168, 255), stop:1 rgba(83,209,236, 255));
  border-radius: 10px;

  box-shadow: 0 -3px rgb(53,167,110) inset;
  transition: 0.6s;

  box-shadow: 0 3px rgb(33,147,90) inset;</string>
   </property>
   <property name="text">
    <string>Deal</string>
   </property>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>600</y>
     <width>250</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1000 $</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pCard1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>560</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>560</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>560</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCard7">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard6">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dCard7">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>143</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>143</width>
     <height>200</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Score2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>661</y>
     <width>731</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="coin5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/pictures/coins/5.png</normaloff>:/img/pictures/coins/5.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="coin20">
      <property name="styleSheet">
       <string notr="true">border-radius: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/pictures/coins/20.png</normaloff>:/img/pictures/coins/20.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="coin50">
      <property name="styleSheet">
       <string notr="true">border-radius: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/pictures/coins/50.png</normaloff>:/img/pictures/coins/50.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="coin100">
      <property name="styleSheet">
       <string notr="true">border-radius: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/pictures/coins/100.png</normaloff>:/img/pictures/coins/100.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="coin500">
      <property name="styleSheet">
       <string notr="true">border-radius: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/img/pictures/coins/500.png</normaloff>:/img/pictures/coins/500.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="dScore">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Score">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pScore">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>580</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bet">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="betValue">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>250</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="hit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(255, 249, 202);
  background:qlineargradient(spread:pad, x1:0.523, y1:0.932, x2:0.522818, y2:0.136, stop:0 rgba(33, 158, 27, 255), stop:1 rgba(180, 255, 82));
  border-radius: 10px;

  box-shadow: 0 -3px rgb(53,167,110) inset;
  transition: 0.6s;

  box-shadow: 0 3px rgb(33,147,90) inset;</string>
   </property>
   <property name="text">
    <string>Hit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doubleX2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>360</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(255, 249, 202);
  background:qlineargradient(spread:pad, x1:0.523, y1:0.932, x2:0.522818, y2:0.136, stop:0 rgba(33, 158, 27, 255), stop:1 rgba(180, 255, 82));
  border-radius: 10px;

  box-shadow: 0 -3px rgb(53,167,110) inset;
  transition: 0.6s;

  box-shadow: 0 3px rgb(33,147,90) inset;</string>
   </property>
   <property name="text">
    <string>Double x2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stand">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(255, 249, 202);
  background:qlineargradient(spread:pad, x1:0.523, y1:0.932, x2:0.522818, y2:0.136, stop:0 rgba(33, 158, 27, 255), stop:1 rgba(180, 255, 82));
  border-radius: 10px;

  box-shadow: 0 -3px rgb(53,167,110) inset;
  transition: 0.6s;

  box-shadow: 0 3px rgb(33,147,90) inset;</string>
   </property>
   <property name="text">
    <string>Stand</string>
   </property>
  </widget>
  <widget class="QLabel" name="endBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>1200</width>
     <height>280</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/pictures/background/endBg.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="link">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>340</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Niagara Solid</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(202, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="split">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>280</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(255, 249, 202);
  background:qlineargradient(spread:pad, x1:0.523, y1:0.932, x2:0.522818, y2:0.136, stop:0 rgba(33, 158, 27, 255), stop:1 rgba(180, 255, 82));
  border-radius: 10px;

  box-shadow: 0 -3px rgb(53,167,110) inset;
  transition: 0.6s;

  box-shadow: 0 3px rgb(33,147,90) inset;</string>
   </property>
   <property name="text">
    <string>Split</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
